src/include/rados.h

```diff
@@ -118,6 +118,7 @@ struct ceph_eversion {
 #define CEPH_OSD_NEARFULL (1<<5) /* osd is at or above nearfull threshold */
 #define CEPH_OSD_BACKFILLFULL (1<<6) /* osd is at or above backfillfull threshold */
 #define CEPH_OSD_DESTROYED (1<<7) /* osd has been destroyed */
+#define CEPH_OSD_DESTROYING (1<<8) /* osd is being destroyed */
```

Comment:
I suppose we could keep the state of the op as a key in the 'mon' prefix, and add it as we start the op, and remove it with the last transaction. Yeah. Let me do that.